service/urls.py

```diff
@@ -23,4 +23,6 @@
     url(r'^posts/(?P<pk>[0-9a-z\\-]+)/', service.posts.views.AllPostsViewSet.as_view({"get": "retrieve"}),
         name='post-detail'),
     url(r'^author/(?P<pk>[0-9a-z\\-]+)/posts/', service.posts.views.AuthorPostsList.as_view(), name='author-posts-list'),
+    url(r'^author/(?P<me>[0-9a-z\\-]+)/friends/(?P<you>[0-9a-z\\-]+)',
```

Comment:
It's not entirely clear whether that endpoint is also required as the specification suggests it is optional. This API implementation is based on what the specification describes on [line 205](https://github.com/abramhindle/CMPUT404-project-socialdistribution/blob/master/example-article.json#L205). Any thoughts on that, Adam?